core/src/main/java/com/redhat/lightblue/client/LightblueClientConfiguration.java

```diff
@@ -6,6 +6,21 @@
 
 public class LightblueClientConfiguration {
 
+    public enum Compression {
+        NONE, LZF;
+
+        public static Compression parseCompression(String str) {
+            switch (str) {
+            case "NONE":
+                return NONE;
+            case "LZF":
+                return LZF;
+            default:
+                throw new IllegalArgumentException("Invalid compression " + str + ". Supported values are: NONE, LZF");
```

Comment:
What happens if the client advertises the ability, but NONE was set on the server? I assume lzf would not be used. Would the client still be able to read the response? Or is it expecting lzf to be returned?